internal/output/combined.go

```diff
@@ -84,6 +86,40 @@ func LoadPaths(paths []string) ([]ReportInput, error) {
 	return inputs, nil
 }
 
+// CompareTo generates an output Root using another Root as the base snapshot.
+// Each project in current Root will have all past resources overwritten with the matching projects
+// in the prior Root. If we can't find a matching project then we assume that the project
+// has been newly created and will show a 100% increase in the output Root.
+func CompareTo(current, prior Root) (Root, error) {
+	priorProjects := make(map[string]*schema.Project)
+	for _, p := range prior.Projects {
+		priorProjects[p.Name] = p.ToSchemaProject()
```

Comment:
Should we error if there are multiple prior projects with the same name?

Reply:
yeah probably

Reply:
fixed in https://github.com/infracost/infracost/pull/1570/commits/b17b88f6110bbb15ecfe939c6a3d55e5ba5d74f9